catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/events/EventApplication.java

```diff
@@ -0,0 +1,77 @@
+package org.codice.ddf.catalog.ui.events;
+
+import static spark.Spark.get;
+
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.servlet.SparkApplication;
+
+public class EventApplication implements SparkApplication {
+
+  private static final List<PrintWriter> listeners = new ArrayList<>();
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(EventApplication.class);
+
+  @Override
+  public void init() {
+    get(
+        "/events",
+        (req, res) -> {
+          try {
+            res.type("text/event-stream; charset=UTF-8");
+            res.header("Connection", "keep-alive");
+            res.header("Cache-Control", "no-cache");
+            res.status(200);
+            PrintWriter out = res.raw().getWriter();
+            synchronized (listeners) {
+              listeners.add(out);
+            }
+            out.write(": \n\n");
+            out.flush();
+
+            while (true) {
+              // Sending SSE heartbeat
+              out.write(": \n\n");
+              if (out.checkError()) {
+                // Subscriber error, break out of loop
+                break;
+              }
+              Thread.sleep(1000);
+            }
+            listeners.remove(out);
+            return "";
```

Comment:
I'm not sure what I should be returning here (and on line 51). we're not using these return values at all. @bdeining any thoughts?